upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12-v1.9.yaml.template

```diff
@@ -958,5 +983,72 @@ spec:
           - key: config.yaml
             path: config.yaml
         name: config
+      - projected:
+          sources:
+          - secret:
+              name: hubble-relay-client-certs
+              items:
+                - key: tls.crt
+                  path: client.crt
+                - key: tls.key
+                  path: client.key
+                - key: ca.crt
+                  path: hubble-server-ca.crt
+          - secret:
+              name: hubble-relay-server-certs
+              items:
+                - key: tls.crt
+                  path: server.crt
+                - key: tls.key
+                  path: server.key
+        name: tls
+---
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
Shouldn't this be v1 already?

Reply:
ouch yes. not sure how v1alpha2 made it there.  thanks for cat